client/src/components/FormsUI/Select/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TextField, MenuItem } from '@material-ui/core';
+import { useField, useFormikContext } from 'formik';
+
+const SelectWrapper = ({ name, options, ...otherProps }: any): JSX.Element => {
+  const { setFieldValue } = useFormikContext();
+  const [field, meta] = useField(name);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    setFieldValue(name, value);
+  };
+
+  const configSelect = {
+    ...field,
+    ...otherProps,
+    select: true,
+    variant: 'outlined',
+    fullWidth: true,
+    onChange: handleChange,
+  };
+
+  if (meta && meta.touched && meta.error) {
+    configSelect.error = true;
+    configSelect.helperText = meta.error;
+  }
+
+  return (
+    <TextField {...configSelect}>
+      {options.map((item: any, pos: any) => {
```

Comment:
In order to be more flexible, I would recommend accepting these from the `children` prop instead. There are instances where we will want to provide different `value`s from the item text in our select menu items as well as being able to pass `string`s or `number`s as values. E.g.
```
<MenuItem value={10}>Ten</MenuItem>
<MenuItem value={20}>Twenty</MenuItem>
<MenuItem value={30}>Thirty</MenuItem>
```
It's difficult to provide the same functionality with a different implementation as far as I can think of 👍🏼

Same comments as the custom TextField component apply here as well.

Reply:
> 

@ugur-akin I seem lost with this comment, if you could elaborate it please